src/helpers/cliUtils.js

```diff
@@ -128,6 +129,43 @@ async function extractDataForPatients(auth, config, patientIds, icareClient, mes
   return errors;
 }
 
+async function sendEmailNotification(notificationInfo, errors) {
+  const totalErrors = Object.keys(errors).reduce((previousValue, currentValue) => previousValue + errors[currentValue].length, 0);
+  if (totalErrors === 0) {
+    return;
+  }
+
+  if (!notificationInfo.from || !notificationInfo.to || !notificationInfo.host || !notificationInfo.port) {
+    throw new Error('Notification information incomplete. Unable to send email.');
+  }
+
+  // Aggregate errors and build email message
+  let emailBody = 'Thank you for using the mCODE Extraction client, provided by the STEAM team. ';
+  emailBody += 'Unfortunately, the following errors occurred when running the extraction client:\n\n';
+  Object.keys(errors).forEach((patientRow) => {
+    emailBody += `Errors for patient at row ${parseInt(patientRow, 10) + 1} in .csv file:\n\n`;
+    errors[patientRow].forEach((e) => { emailBody += `${e.message.trim()}\n`; });
+    if (errors[patientRow].length === 0) {
+      emailBody += 'No errors for this patient. Extraction was successful.\n';
+    }
+    emailBody += '\n============================================================\n\n';
+  });
+  emailBody += 'For more information about these errors, run the extraction client using the `--debug` flag.';
```

Comment:
DP3 - This would be for more real-time information, right? Could we update the message accordingly? Don't want people thinking that there's more information they'd see in this email if they ran the program with a flag properly set.

Reply:
Correct! I can clarify!

Reply:
Actually, I could include it in the email if that seems helpful. Do you think that would be helpful?

Reply:
Talked offline and it sounded like it would be helpful. So now if the `--debug` flag is used in the cli, the addition stack trace info that appears in the terminal also is included in the email. If this is actually confusing, I can change it back easily.